app/javascript/modules/listingDetails/ListingDetailsHMITable.tsx

```diff
@@ -216,28 +235,7 @@ export const ListingDetailsHMITable = ({ listing }: ListingDetailsEligibilityPro
   }
 
   return (
-    <ListSection
-      title={t("listings.householdMaximumIncome")}
-      subtitle={
-        <div>
-          <div className="mb-4">{renderMarkup(t("listings.forIncomeCalculations"))}</div>
-          <div className="mb-4 primary-lighter-markup-link-desktop">
-            {renderMarkup(
-              t("listings.incomeExceptions.intro", {
-                url: getSfGovUrl(
-                  "https://sf.gov/information/special-calculations-household-income",
-                  7080
-                ),
-              })
-            )}
-          </div>
-          <ul className="list-disc ml-5">
-            <li>{t("listings.incomeExceptions.students")}</li>
-            <li>{t("listings.incomeExceptions.nontaxable")}</li>
-          </ul>
-        </div>
-      }
-    >
+    <span>
```

Comment:
Could we use a fragment here instead `<>...</>`, `table` elements should not be nested in `span` elements.